frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -873,18 +879,22 @@ export const insightLogic = kea<insightLogicType>([
         },
         startQuery: ({ queryId }) => {
             actions.markInsightTimedOut(null)
-            actions.marktInsightErrored(null)
+            actions.markInsightErrored(null)
             values.timeout && clearTimeout(values.timeout || undefined)
             const view = values.activeView
             actions.setTimeout(
                 window.setTimeout(() => {
-                    if (values && view == values.activeView) {
-                        actions.markInsightTimedOut(queryId)
-                        const tags = {
-                            insight: values.activeView,
-                            scene: sceneLogic.isMounted() ? sceneLogic.values.scene : null,
+                    try {
+                        if (values && view == values.activeView) {
+                            actions.markInsightTimedOut(queryId)
+                            const tags = {
+                                insight: values.activeView,
+                                scene: sceneLogic.isMounted() ? sceneLogic.values.scene : null,
+                            }
+                            posthog.capture('insight timeout message shown', tags)
                         }
-                        posthog.capture('insight timeout message shown', tags)
+                    } catch (e) {
```

Comment:
Same here, I think this is only an error in tests. Lazy to wrap it like this but 🤷 🙈